src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -80,6 +81,7 @@ public function action(
         Device $deviceForLocalFiles,
         Usage $queueForUsage
     ): void {
+        Runtime::setHookFlags(SWOOLE_HOOK_ALL ^ SWOOLE_HOOK_TCP);
```

Comment:
We can do this outside the class definition right? Don't think we need to it for every job

Reply:
My concern with putting it outside of the action is that it would get triggered in other workers. For example, I added `Console::warning('Messaging class');` outside the class and it was executed in the build worker:

<img width="379" alt="image" src="https://github.com/appwrite/appwrite/assets/1477010/d700e04a-33fe-4eda-89e9-731f1d8c4c61">

Reply:
Ohh makes sense because they're all initialized together. All good :+1: